sdk/helper/scaleutils/nodeselector/defualt_test.go

```diff
@@ -0,0 +1,40 @@
+package nodeselector
+
+import (
+	"testing"
+
+	"github.com/hashicorp/nomad/api"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_newestCreateIndexClusterScaleInNodeSelectorName(t *testing.T) {
```

Comment:
if this is actually a test for `newest create index`, maybe we should go ahead and call the strategy that?

Reply:
I should rename this test; the strategy itself makes no effort to ensure the description is correct which I why I used default. I just missed this. thanks.